FeatherCoin/Feathercoin
# commits from bitcoin/master

qa: Normalize executable location
https://github.com/bitcoin/bitcoin/pull/13051

[tests] Make rpcauth.py testable and add unit tests
https://github.com/bitcoin/bitcoin/pull/13056

test: add rpcauth-test to AC_CONFIG_LINKS to fix out-of-tree make check
https://github.com/bitcoin/bitcoin/pull/13073

[qt] [tests] Clarify address book error messages, add tests
https://github.com/bitcoin/bitcoin/pull/12830

Tests: don't test against min relay fee information in mining_prioritisetransaction.py
https://github.com/bitcoin/bitcoin/pull/13082

Build txindex in parallel with validation
https://github.com/bitcoin/bitcoin/pull/13033
